testnet/stacks-node/src/burnchains/bitcoin_regtest_controller.rs

```diff
@@ -1480,6 +1481,16 @@ impl BurnchainController for BitcoinRegtestController {
         Ok(())
     }
 
+    fn get_stacks_epochs(&self) -> Vec<StacksEpoch> {
+        match self.config.burnchain.epochs.as_ref() {
+            Some(epochs) => epochs.clone(),
+            None => {
+                let (_, indexer) = self.setup_indexer_runtime();
+                indexer.get_stacks_epochs()
+            }
+        }
+    }
```

Comment:
While I think this is always the same, I think we should just always consult the indexer for the stacks epochs: it should always be configured with the correct epochs from either the config or the static variables.

Reply:
Yup, fixed